test/functional/admin_controller_test.rb

```diff
@@ -282,42 +282,44 @@ def setup
   end
 
   test 'update error_grouping_timeout' do
-    with_config_value(:error_grouping_enabled, true) do
-      with_config_value(:error_grouping_timeout, 1.minute) do
-        post :update_features_enabled, params: { error_grouping_timeout: '1' }
-        assert_equal 1.seconds, Seek::Config.error_grouping_timeout
-        post :update_features_enabled, params: { error_grouping_timeout: '10 sec' }
-        assert_equal 10.seconds, Seek::Config.error_grouping_timeout
-        post :update_features_enabled, params: { error_grouping_timeout: '2 min' }
-        assert_equal 120.seconds, Seek::Config.error_grouping_timeout
-      end
+    # Checks for warnings when not setting error_grouping_timeout
+    with_config_value(:filtering_enabled, 'True') do
+      post :update_features_enabled, params: { filtering_enabled: 'False' }
```

Comment:
the form would send `'0'` here, but not critical as anything but '1' is resolved to false

Reply:
Resolved in [3fb7abb](https://github.com/seek4science/seek/pull/1379/commits/3fb7abb9257b420c6610811d70e742a2a30ca7b1).